gravitee-apim-console-webui/src/management/api/component/health-check-v4-form/api-health-check-v4-form.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card class="healthCheckV4">
+  <div class="apiHealthCheck__header">
```

Comment:
BEM convention recommend to use kebab case for a block and separate blocks by __ so I would change the class name with the suggestion below. 
```suggestion
<mat-card class="health-check">
  <div class="health-check__header">
      ....
```

This comment is true for all the classes in this file for example we can changeb the class on line 25 with something like `health-check__enable-field`